website/docs/r/team_members.html.markdown

```diff
@@ -0,0 +1,45 @@
+---
+layout: "tfe"
+page_title: "Terraform Enterprise: tfe_team_members"
+sidebar_current: "docs-tfe-team-members"
+description: |-
+  Add or remove a users from a team.
+---
+
+# tfe_team_members
+
+Add or remove a users from a team.
+
+~> NOTE on managing team memberships: Terraform currently provides two resources
```

Comment:
_minor_ I'd recommend updating `NOTE` to be `**NOTE:**` to make it stand out

Reply:
Fixed in next commit.